cowprotocol/cowswap
# feat(twap): add TWAP unlock screen

# Summary

Fixes #2656

- adds unlock screen to advanced orders
- refactors current component from limit orders so that its reused

# To test
- delete local storage
- go to advanced orders and limit orders
- there should be unlock screen
- it should work properly
